Fix obsoletes for openSUSE Leap Root on ZFS installation man (#293)
openzfs/openzfs-docs: docs/Getting Started/openSUSE/openSUSE Leap Root on ZFS.rst

if multiple disks (mirror or raidz topologies) are used, the data has to be
encrypted once per disk.

Notes
~~~~~~~

> - You can use unofficial scripts-pack `LroZ <https://github.com/ndruba/LroZ>`__ (Linux Root On Zfs), based on this manual and automated most steps. 

Reviewer: ```suggestion
- You can use unofficial scripts `LroZ <https://github.com/ndruba/LroZ>`__ (Linux Root On Zfs), which is based on this manual and automates most steps. 
```

       sgdisk     -n4:0:0        -t4:8309 $DISK

   **Hints:**

>    - If you want to use only legacy or only UEFI booting, you do not have nessecary to do both partition, only legacy or only UEFI, respectively.

Reviewer: We didn't have issues with obviousness of one-only partition for boot types, let's drop this hint.

>    - For grub 2.04 (Leap 15.3) not worked creating only boot pool with some features disabled. root pool also required some features disabling or ``grub2-install`` was not worked correctly. Because of this, creating separately pool for ``/boot`` directory dont have a reasons. 

Reviewer: Let's drop it too, you stated it in next changes.

   - The pool name is arbitrary. If changed, the new name must be used
     consistently. On systems that can automatically install to ZFS, the root
     pool is named ``rpool`` by default.

>    - If you want to use grub bootloader, you must to set::

Reviewer: It's not obvious, user already created root pool. Let's move it into root pool creation before actual command. Grub is usual choice, so it would be great to add it in command for user and add notes why user may want to enable some features if grub won't be used.

   Output must include that languages:

   * C
   * C.UTF-8
   * C.utf8
   * en_US.utf8
   * POSIX

   Find yout locale from `locale -a` commands output then set it with following command.

   .. code-block:: text

>      localectl set-locale LANG=en_US.UTF-8

Reviewer: Locale set is usual step for any distro, why do we need to drop it?

Reviewer: Still actual question.

     zypper install kernel-default kernel-firmware

   **Note:** If you installed `base` pattern, you need to deinstall busybox-grep to install `kernel-default` package.

#. Install ZFS in the chroot environment for the new system::

     zypper install lsb-release

>      zypper addrepo https://download.opensuse.org/repositories/filesystems/`lsb_release -rs`/filesystems.repo

Reviewer: You updated to use $() everywhere else, let's make it consistent.

     passwd

> #. Enable importing bpool (Of cource, if you created them)

Reviewer: Let's drop it, if user did something not by default - user should understand consequences.

Reviewer: My comment is still actual

     echo 'zfs'>> /etc/modules-load.d/zfs.conf

> #. Kernel version of livecd can differ from currently installed version. Than, check kernel version of you new OS::

Reviewer: ```suggestion
#. Kernel version of livecd can differ from currently installed version. Get kernel version of you new OS::
```

   **Note:** If you have had trouble with the grub2 installation, I suggest you use systemd-boot.
   **Note:** If this command don't gives any output, use classic grub.cfg generation with following command:
   ``grub2-mkconfig -o /boot/grub2/grub.cfg``
   

> #. Check, that ``/boot/grub2/grub.cfg`` have the menuentry have a ``root=ZFS=rpool/ROOT/suse`` in a line, like this::

Reviewer: ```suggestion
#. Check, that ``/boot/grub2/grub.cfg`` have the menuentry ``root=ZFS=rpool/ROOT/suse`` in a line, like this::
```

     linux   /boot@/vmlinuz-5.3.18-150300.59.60-default root=ZFS=rpool/ROOT/suse

>    If not, change ``/etc/default/grub``::
>    
>      vi /etc/default/grub
>      # Set: GRUB_CMDLINE_LINUX="root=ZFS=rpool/ROOT/suse"   

Reviewer: ```suggestion
   If not, change ``/etc/default/grub``::
   
    GRUB_CMDLINE_LINUX="root=ZFS=rpool/ROOT/suse"   
```

   compression algorithm can reduce I/O. The exception is all-zero pages,
   which are dropped by ZFS; but some form of compression has to be enabled
   to get this behavior.
   

>    **Hints:** If you saw warning about uncorrect volblocksize, recreate swap device:

Reviewer: ```suggestion
   **Hints:** If you've got a warning about incorrect volblocksize, recreate swap device:
```

Reviewer: Hm, but what's the case when user can get such problems? `getconf PAGESIZE` should work.

#. Wait for the system to boot normally. Login using the account you
   created. Ensure the system (including networking) works normally.

> #. Configure locale: Find your locale from ``locale -a`` commands output then set it with following command::

Reviewer: What's the purpose to move it here?

Reviewer: Still actual question, I don't see a reason to do it after basic system configuration.
